resources/brave-fix.js

```diff
@@ -2,3 +2,5 @@
 /// alias bf.js
 delete Navigator.prototype.brave
 delete window.navigator.brave
+delete navigator.globalPrivacyControl
+delete Navigator.prototype.globalPrivacyControl
```

Comment:
We should not bundle GPC disablement with `window.navigator`. All disablements should be granular.